Makefile

```diff
@@ -27,7 +27,7 @@ FINDFILES=find . \( -path ./.git -o -path ./.github \) -prune -o -type f
 XARGS = xargs -0 ${XARGS_FLAGS}
 CLEANXARGS = xargs ${XARGS_FLAGS}
 
-REGISTRY = quay.io/open-cluster-management
+REGISTRY = quay.io/stolostron
```

Comment:
Makefile is use for community and should stay with `quay.io/open-cluster-management`

Reply:
@mikeshng OK updated